images/kcp-registrar/Dockerfile

```diff
@@ -30,15 +30,16 @@ RUN mkdir /workspace && chmod 777 /workspace && chown 65532:65532 /workspace
 RUN mkdir $HOME && chmod 777 $HOME
 COPY --from=builder workspace/kcp/bin/kubectl-kcp /usr/local/bin/kubectl-kcp
 RUN chmod 755 /usr/local/bin/kubectl-kcp
-COPY ./register.sh /usr/local/bin/register.sh
-RUN chmod 755 /usr/local/bin/register.sh
+RUN microdnf install findutils git
 RUN JQ_VERSION=1.6 && \
-    curl -sSL -o /usr/local/bin/jq https://github.com/stedolan/jq/releases/download/jq-$JQ_VERSION/jq-linux64 && \
+    curl --fail -sSL -o /usr/local/bin/jq https://github.com/stedolan/jq/releases/download/jq-$JQ_VERSION/jq-linux64 && \
     chmod 755 /usr/local/bin/jq
 RUN KUBE_VERSION=v1.24.0 && \
-    curl -L -o /usr/local/bin/kubectl "https://dl.k8s.io/release/$KUBE_VERSION/bin/linux/amd64/kubectl" && \
+    curl --fail -L -o /usr/local/bin/kubectl "https://dl.k8s.io/release/$KUBE_VERSION/bin/linux/amd64/kubectl" && \
     chmod 755 /usr/local/bin/kubectl
+COPY ./register.sh /usr/local/bin/register.sh
+RUN chmod 755 /usr/local/bin/register.sh
 USER 65532:65532
 VOLUME /workspace
 WORKDIR /workspace
-CMD ["/usr/local/bin/register.sh"]
+ENTRYPOINT /usr/local/bin/register.sh
```

Comment:
What is the rational for removing the command and overriding the default entrypoint?
"/bin/sh -c" is convenient for any troubleshooting exercise. Surely it is possible to override that in Kubernetes or with the container run command of the container engine but in my opinion this change makes the image more difficult to use.